src/software/embedded/constants/jetson_constants.h

```diff
@@ -8,6 +8,6 @@ constexpr const char MOTOR_DRIVER_RESET_GPIO[]                = "168";
 constexpr const char DRIVER_CONTROL_ENABLE_GPIO[]             = "194";
 
 // Path to the CPU thermal zone temperature file
-constexpr const char CPU_TEMP_FILE_PATH[] = "/sys/class/thermal/thermal_zone0/temp";
+constexpr const char CPU_TEMP_FILE_PATH[] = "/sys/class/thermal/thermal_zone1/temp";
```

Comment:
I think Arune maybe have copied the wrong file: 

This is the path in upstream/master: 

https://github.com/UBC-Thunderbots/Software/blob/8e45ea06e21b98fd86f7355fedfcb38533a76589/src/software/jetson_nano/thunderloop.h#L140